examples/cli.rs

```diff
@@ -37,8 +40,26 @@ fn main() {
                             .short("r")
                             .long("resize")
                             .help("Enable auto resize")))
+            .subcommand(SubCommand::with_name("iter-quantiles")
+                    .about("Display quantiles to stdout from serialized histogram stdin")
+                    .arg(Arg::with_name("ticks")
+                            .short("t")
+                            .long("ticks-per-half")
+                            .takes_value(true)
+                            .required(true)
+                            .help("Ticks per half distance"))
+                    .arg(Arg::with_name("quantile-precision")
+                            .long("quantile-precision")
+                            .takes_value(true)
+                            .default_value("5")))
             .get_matches();
 
+    let stdin = std::io::stdin();
+    let stdin_handle = stdin.lock();
```

Comment:
Any particular reason you don't just want `let stdin = std::io::stdin().lock();`?

Reply:
The compiler gets grumpy about the lifetime of `stdin()` if I remove the intermediate `let`. Perhaps there is a better way though?

Reply:
Ah.. This is at least a little nicer:
```rust
let stdin = std::io::stdin();
let stdin = stdin.lock();
```